docs/DeveloperGuide.md

```diff
@@ -695,3 +695,77 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Mass Operations [Proposed]
```

Comment:
Would it be better to add the `[Proposed]` tag before the command itself for better consistency?
> ### \[Proposed\] Undo/redo feature